src/core/sys/freebsd/netinet/in_.d

```diff
@@ -0,0 +1,581 @@
+//Written in the D programming language
```

Comment:
Hmm, looks like it's half/half for phobos, and 23 vs 450 for druntime (that's why it looked strange to me), but fair enough :+1: